src/main/groovy/com/palantir/gradle/javadist/JavaDistributionPlugin.groovy

```diff
@@ -58,19 +59,37 @@ class JavaDistributionPlugin implements Plugin<Project> {
             }
         }
 
+        Task copyLauncherBinaries = project.tasks.create('copyLauncherBinaries', {
+            group = GROUP_NAME
+            description = "Creates go-java-launcher binaries."
+        }) << {
+            // TODO(rfink) Don't check these in, but fetch them from a published artifact?
+            ["javalauncher-linux-amd64", "javalauncher-darwin-amd64"].each { file ->
+                def dest = Paths.get("${project.buildDir}/scripts/${file}")
```

Comment:
double-checking the reasoning here: the file is really part of the binary and is not intended to be edited by anyone directly. I think it's more correct to put it in bin/. as it stands, the stuff in the var/conf directory of the distribution is exactly the contents of the var/conf directory of the source, and conversely everything in bin/ is generated; I like that separation.